### ISSUE TITLE
Add config URL parameter to booking system

### ISSUE THREAD

**Issue by timdrysdale** [practable/relay#13]
Problem: We can pass customised streams to UI, and it would now be helpful to pass customised assets like images of experiments that will not be using video.

Solution: Feature request for cfg URL parameter, which identifies a config object written in json that the UI can collect from the assets server, e.g. when passed `config=https://assets.practable.io/config/experiments%2Fpvna%2Fpvna01-0.0.json` 

Information in the config cannot be private/secret because it is neither obfuscated nor protected. It's intended for harmless assets like images. Do not include secrets such as API keys or JWT tokens for accessing other services. 

Implementation details:
We will do this specifically for `config` URL param only:

0. Add key:`config`, value type string, to the activity data structure
0. Extend the bash variable substitution feature in the pkg/book(ing) to be extended to handle a `config` in the 

Note: we could  (spoiler: won't) consider adding the general capability to pass a customised list of "static" URL parameters. Since there is no limit to what can be put in the config file, it seems to duplicate functionality, and split config information into multiple places. It also means the UI to the booking system has to understand general parameters, whereas it would be easier to push that concern to the config file management tool, which will have to understand a wider range of data structures. This way, we keep all config in one place, and avoid configuration conflict. We structure the folders in the `assets/config` to support versioning.

Example config token (note that each UI would be free to follow it's own preferred extended format

```
{
    "name":"pvna01",
    "version":"0.0",
    "date":1644326235,
    "images":{
        "dut":{
            "url":"https://static.practable.io/assets/images/experiments/pvna01/dut.png",
            "size":{
                "width":"300px",
                "height":"480px"
            },
            "sparam":{
                "url":"https://static.practable.io/assets/images/experiments/pvna01/sparam.svg",
                "size":{
                    "width":"300px",
                    "height":"480px"
                }
            },
            "parameters":{
                "dut":{
                    "name":"80mm Antenna",
                    "d1":"80mm",
                    "d2":"10mm",
                    "d3":"10mm",
                    "d4":"15mm",
                    "w":"3.5mm",
                    "c1":"1pF",
                    "c2":"2pF"
                },
                "board":{
                    "material":"FR4",
                    "thickness":"1.6mm"
                }
            }
        }
    }
}

```

**Comment by timdrysdale** [comment]
Also, it would be preferable to sanitise the `config` URL parameter passed to the UI so there is consistently no leading slash, even if the manifest has one. The UI is responsible for sanity checking anyway, so it can handle a leading / in the parameter, but it will make the URL parameter more readable during troubleshooting if there is no additional %2F (escaped value of/)

**Comment by timdrysdale** [comment]
Solved with #14
